### PR TITLE
Bug 1142104 - Progress bar now announces loading/loaded

### PR DESCRIPTION
**Pull request by funktr0n**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1b18064a7bb741a47c4b6e298567109ca6bbc99c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=82ff111c554595350f07a4c413c133c97f6cc791

File: apps/system/js/app_chrome.js [hunk 1]
```diff
@@ -110,7 +110,8 @@
     var className = this.CLASS_NAME + this.instanceID;
 
     return `<div class="chrome chrome-combined" id="${className}">
-              <gaia-progress></gaia-progress>
+              <gaia-progress role="progressbar" aria-live="polite">
```
Comment (eeejay) [inline apps/system/js/app_chrome.js]: I would have these attributes set automatically in the bootstrap of gaia-progress.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=20910d817a6e9c3b52e005b0e3376cfedbdb0111

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=da2a003a6190d0eff4641920cf985207e19425bd

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8a5318d0fd53361aed5527f8a7635ea89f3dea42

**Pull request merged**